cli/print/logger.go

```diff
@@ -0,0 +1,233 @@
+package print
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/mattn/go-isatty"
+	"github.com/tj/go-spin"
+)
+
+type Logger struct {
+	w io.Writer
+	spinnerStopCh chan bool
+	spinnerMsg    string
+	spinnerArgs   []interface{}
+	isSilent      bool
+	isVerbose     bool
+}
```

Comment:
why write our own UI lib here?

Reply:
as discussed, poached from kots but needed to fork it so it could support a custom `io.Writer` https://github.com/replicatedhq/kots/blob/master/pkg/logger/logger.go